local/artena/externallib.php

```diff
@@ -1333,9 +1390,109 @@ public static function remove_enrol_returns() {
         return null;
     }
 
+
```

Comment:
This section was included in plugin obtained from SMSS, however, wasn't included in GitHub